browser/admin/src/AdminSocketSettings.js

```diff
@@ -106,7 +106,7 @@ var AdminSocketSettings = AdminSocketBase.extend({
 			var lokitVersionObj = JSON.parse(textMsg.substring(textMsg.indexOf('{')));
 			h = lokitVersionObj.BuildId.substring(0, 10);
 			if (parseInt(h,16).toString(16) === h.toLowerCase().replace(/^0+/, '')) {
-				h = '<a target="_blank" href="https://hub.libreoffice.org/git-core/' + h + '">' + h + '</a>';
+				h = '<a target="_blank" href="https://git.libreoffice.org/core/+log/' + lokitVersionObj.BuildId + '/">' + h + '</a>';
```

Comment:
Nit: constructing the HTML like this is always dangerous (in case you forget to escape something), but probably enough to fix it up when we get rid of jquery here anyway.